applications/crossbar/src/modules/cb_recordings.erl

```diff
@@ -68,8 +64,8 @@ init() ->
 -spec allowed_methods(path_token(), path_token()) -> http_methods().
 
 allowed_methods() -> [?HTTP_GET].
-allowed_methods(_RecordingId ) -> [?HTTP_GET].
-allowed_methods(_RecordingId, ?ATTACHMENT) -> [?HTTP_GET].
+allowed_methods(_) -> [?HTTP_GET].
+allowed_methods(_, _) -> [?HTTP_GET].
```

Comment:
if `?ATTACHMENT` is not being used, we should remove `allowed_methods/2` , same for other methods